src/Host/Broker/Impl/Resources.resx

```diff
@@ -255,4 +255,7 @@
   <data name="Error_ExitRHost" xml:space="preserve">
     <value>RHost Exited with Win32 error : {0}</value>
   </data>
+  <data name="Text_ServiceName" xml:space="preserve">
+    <value>RHostBrokerService</value>
```

Comment:
We don't use terms 'broker' or 'host' in user-facing messages since they are specific to our implementation and user has no idea what do they mean